cairo/surface.c

```diff
@@ -305,7 +319,7 @@ surface_set_device_offset (PycairoSurface *o, PyObject *args) {
   double x_offset, y_offset;
 
   if (!PyArg_ParseTuple (args, "dd:Surface.set_device_offset",
-			 &x_offset, &y_offset))
+                         &x_offset, &y_offset))
```

Comment:
What's with all the whitespace changes?

Reply:
There was a mix of tabs and spaces used for indentation, which I assumed where done by accident.